### ISSUE TITLE
Structure and format

### ISSUE THREAD

**Issue by Hucxley** [koltclassic/bttgenerator#1]
I'm looking over the work so far, and I think that we need to change the word "questions" to "statements", "prompts", "description" - something more obvious for someone using dot notation.  Probably description, honestly.  Also, we need to get rid of the "..." at the end of each prompt, and replace it with a single space, and swap the upper case at the beginning of each option for a lower case letter.  That way, it reads like this: var descSelector = Dungeons.Castles.descriptions.id: [Math.floor(Math.random(0, Dungeons.Castles.description.length()-1)+1)]; var descSubject = descSelector.subject; var descPredicate = descSelector.predicate[Math.floor(Math.random(0,descSelector.dieToRoll-1)+1)]; var fullStatement = descSubject + descPredicate;

I've made some changes to the file you currently have up.  If you like the direction I'm going, let me know after you accept the PR, otherwise, I'll leave yours alone and go make a different one that doesn't stomp all over your data model.

**Comment by koltclassic** [comment]
There aren't any PRs showing up for me to accept, but once you send it I'll be sure to accept it.

**Comment by Hucxley** [comment]
Yeah, I tried to create one last night, but it was late and github was
prompting me for a branch to make the PR on. I probably need to go back and
fork it first, then move my changes to the fork, then make the PR. I
usually work on my own stuff, so I'm rusty on contributing side of open
source projects. I did clean up all of what was already done and added the
other main categories of Dungeons (no details, though). I'll mess around
with it all tonight after work and get a PR pushed to you.
